.github/workflows/kitchen-tests.yml

```diff
@@ -87,81 +85,81 @@ jobs:
     container:
       image: "${{ needs.docker.outputs.full_image_id }}"
     steps:
-    - uses: actions/checkout@v2
-      with:
-        clean: false #this is to avoid permissions on self-hosted runner
-    - name: Pre-check dirty dirs in self-hosted 
-      run: |
-        ls -la ~
-        ls -la .
-        ls -la examples/sample-jhub
-        ls -la examples/sample-jhub-nfs
-        kubectl config view
-    - name: Remove Kitchen File
-      run: |
-        rm -rf .kitchen
-        rm -rf .terraform
-        rm -rf examples/*/.terraform
-        rm -rf examples/*/terraform.tfstate*
-        rm -f ~/.kube/config
-        ls -la ~/.kube || :
-    - name: Post-check dirty dirs in self-hosted
-      run: |
-        ls -la ~
-        ls -la .
-        ls -la examples/sample-jhub
-        ls -la examples/sample-jhub-nfs
-    - name: Create Credential File
-      run: |
-        echo "$GCP_CREDENTIAL_JSON" | base64 -d > /tmp/credentials.json
-      env:
-        GCP_CREDENTIAL_JSON: ${{ secrets.GCP_CREDENTIAL_JSON}}
-    - name: Create TLS Files
-      run: |
-        echo "$JUPYTERHUB_TLS_CER" > /tmp/tls.cer
-        echo "$JUPYTERHUB_TLS_KEY" > /tmp/tls.key
-      env:
-        JUPYTERHUB_TLS_CER: ${{ secrets.JUPYTERHUB_TLS_CER }}
-        JUPYTERHUB_TLS_KEY: ${{ secrets.JUPYTERHUB_TLS_KEY }}
-    - name: Authorize service account
-      run: | 
-        gcloud auth activate-service-account ${{ secrets.GCP_PF_SA }} --key-file=/tmp/credentials.json
-        kubectl config view
-        ls -la /tmp
-    - name: Run Kitchen
-      run: kitchen test
-      env:
-        TF_VAR_billing_account: ${{ secrets.GCP_BILLING_ACCOUNT }}
-        TF_VAR_org_id: ${{ secrets.GCP_ORG_ID }}
-        TF_VAR_folder_id: ${{ secrets.GCP_CCV_CI_FOLDER_ID }}
-        INFOBLOX_USERNAME: ${{ secrets.INFOBLOX_JHUB_USER }}
-        INFOBLOX_PASSWORD: ${{ secrets.INFOBLOX_JHUB_PSWD }}
-        INFOBLOX_SERVER: ${{secrets.INFOBLOX_JHUB_HOST}}
-        TF_VAR_site_certificate_file: /tmp/tls.cer
-        TF_VAR_site_certificate_key_file: /tmp/tls.key
-        USE_GKE_GCLOUD_AUTH_PLUGIN: true
-    - name: Clean kitchen and terraform files
-      run: |
-        rm -rf .kitchen
-        rm -rf examples/*/.terraform
-        rm -rf examples/*/terraform.tfstate*
-        rm -f ~/.kube/config
-        ls -la ~/.kube || :
-    - name: Post-check dirty dirs in self-hosted
-      run: |
-        ls -la .
-        ls -la examples/sample-jhub
-        ls -la examples/sample-jhub-nfs
-        ls -la ~
-  
-  # Clean up dangling images in self-hosted runner to avoid filling up docker disk 
+      - uses: actions/checkout@v3
+        with:
+          clean: false #this is to avoid permissions on self-hosted runner
+      - name: Pre-check dirty dirs in self-hosted
```

Comment:
lets do this on another PR, but our self-hosted runners are now ephemeral and all of this checking/cleaning of files and directories can be removed

Reply:
Sounds great! Will add another PR to address this